modules/efk-logging-curator-cron.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * logging/efk-logging-curator.adoc
+
+[id='efk-logging-curator-cron_{context}']
+= Configuring Curator using a CronJob
+
+You can alternatively create, configure, and run Curator using a CronJob. The next scheduled job uses this configuration.
+
+You can use cronjob for debugging Curator. For example, if curator is in failed state, but the log messages do not 
+provide a reason, you could increase the log level and run the cronjob with this command, instead of waiting for another scheduled run of the cron job.
+
+.Prerequisite
+
+Cluster logging and Elasticsearch must be installed.
+
+.Procedure
+
+To create a cronjob for Curator:
+
+. Run the following command to create cronjob:
++
+----
+$ oc create job --from=cronjob/curator <job_name>
+----
++
+. Create and modify the configuration:
++
+* For scripted deployments
++
+.. Copy the Curator configuration file and the {produxt-title} custom configuration file 
+from the Curator configuration map and create separate files for each:
++
+----
+$ oc extract configmap/curator --keys=curator5.yaml,config.yaml --to=/my/config
+----
++
+.. Edit the *_/my/config/curator5.yaml_* and *_/my/config/config.yaml_* files.
++
+.. Delete the existing Curator config map and add the edited YAML files to a new Curator config map.
++
+----
+$ oc delete configmap curator ; sleep 1
+$ oc create configmap curator \
+    --from-file=curator5.yaml=/my/config/curator5.yaml \
+    --from-file=config.yaml=/my/config/config.yaml \
+    ; sleep 1
+----
++
+* Alternatively, if you are using the *actions file*:
++
+.. Copy the Curator configuration file, the {produxt-title} custom configuration file, and the
+the {produxt-title} *action* file from the Curator configuration map and create separate files for each:
++
+----
+$ oc extract configmap/logging-curator --keys=curator5.yaml,actions.yaml --to=/my/config
+----
++
+.. Edit the *_/my/config/curator5.yaml_* and *_/my/config/config.yaml_* files.
++
+.. Delete the existing Curator config map and add the edited YAML files to a new Curator config map.
++
+----
+$ oc delete configmap curator ; sleep 1
+$ oc create configmap curator \
+    --from-file=curator5.yaml=/my/config/curator5.yaml \
+    --from-file=config.yaml=/my/config/config.yaml \
```

Comment:
should be : --from-file=actions.yaml=/my/config/actions.yaml \

Reply:
@QiaolingTang Thank you. Time to stop for the day. Making errors from being tired!